packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -93,6 +94,10 @@ class CupertinoPageScaffold extends StatelessWidget {
       if (fullObstruction) {
         paddedContent = MediaQuery(
           data: existingMediaQuery.copyWith(
```

Comment:
There's another API you can use:

```dart
existingMediaQuery.removePadding(removeTop: true),
```

Saves you some typing and is a bit more semantic. Keep the comment below though.